package.json

```diff
@@ -0,0 +1,25 @@
+{
+  "name": "explorer.loopback.io",
+  "version": "1.0.0",
+  "description": "Scripts for the jekyll-based explorer.loopback.io site",
```

Comment:
> Scripts for the jekyll-based explorer.loopback.io site

The package description is out of sync with the actual implementation.

Reply:
Great catch. Open #4 to fix.